stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -994,18 +996,19 @@ internal constructor(
     //endregion
 
     public fun disconnectSocket() {
-        chatSocket.disconnect()
+        launch { chatSocket.disconnect() }
     }
 
     public fun reconnectSocket() {
```

Comment:
Not related to this ticket.
What do you think about aligning `disconnectSocket` and `reconnectSocket` signature with other functions in `ChatClient`?
```kotlin
public fun disconnectSocket(): Call<Unit> {
    //...
}

public fun reconnectSocket(): Call<Unit> {
    //...
}
```